### ISSUE TITLE
Doesn't seem to work for me?

### ISSUE THREAD

**Issue by amenk** [cmuench/phpstorm-downloader#2]
```
foo@bar:/opt/phpstorm-downloader$ sudo bin/phpstorm-downloader download /optRequest
PhpStorm EAP: OK
foo@bar:/opt/phpstorm-downloader$ 
```

All this happens in less than a second - nothing seems to be downloaded

**Comment by ktomk** [comment]
Looks like a parsing problem for the EAP version. Could have been the case when there is no EAP version available (or the parsing does not work with the current EAP wiki page).

I address a similar problem with #3:
- Fix EAP version parsing.
- New option `--stable` to download the latest stable version.

The latter should help in case there is no EAP version.

**Comment by ktomk** [comment]
This is fixed, also in #6. In case this happens again, you can use -v and -vvv to obtain more information. This normally signals URL changes for the EAP version.
